library/std/src/fs/tests.rs

```diff
@@ -1534,3 +1534,14 @@ fn read_large_dir() {
         entry.unwrap();
     }
 }
+
+#[test]
+#[cfg(windows)]
+fn hiberfil_sys() {
+    // Get the system drive, which is usually `C:`.
```

Comment:
I thought Windows has `C:` hard coded in various places and as such effectively requires the system drive to be `C:`.

Reply:
Pretty much. I didn't want to make any assumptions but I am probably being overly cautious in this case.